instance/tests/integration/ansible/playbooks/failignore.yml

```diff
@@ -12,6 +12,17 @@
 
 - name: Fail and ignore on purpose
   hosts: all
+  become: true
   tasks:
+    - file:
```

Comment:
Both the ansible_failure and ansible_failignore tests failures in running playbooks and how we deal with those in code. We actually provision a new test instance and appserver for each, so there should be no conflicts, unless I misunderstood the comments. I don't know why they run `apt` but I guess it's simply testing the VMs are working fine and we can leave that.

Due to the unexpected complexity mocking this, our constant improvements to the integration tests and the fact this has been tested working, I'd say we keep this behavior.

Reply:
> I don't know why it updates packages through apt. I don't fully understand the script so I might be wrong.
> I don't know why they run apt but I guess it's simply testing the VMs are working fine and we can leave that.

`python` is required so that ansible can run any commands besides apt on the hosts. So the script runs `apt update` and then installs `python-minimal`. :)